cmd/samInfo/readLength_test.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/fileio"
+	"testing"
+)
+
+var ReadLengthTests = []struct {
+	InFile       string
+	OutFile      string
+	ExpectedFile string
+}{
+	{InFile: "testdata/readLength/small.sam",
+		OutFile:      "testdata/readLength/test.readLength.txt",
+		ExpectedFile: "testdata/readLength/expected.readLength.txt",
+	},
+}
+
+func TestReadLength(t *testing.T) {
+	var s ReadLengthSettings
+	for _, v := range ReadLengthTests {
+		s = ReadLengthSettings{
+			InFile:  v.InFile,
+			OutFile: v.OutFile,
+		}
+		readLength(s)
+		if !fileio.AreEqual(v.OutFile, v.ExpectedFile) {
```

Comment:
Directly test the read length functionality rather than reading in expected test files